Prace_domowe/Praca_domowa2/Grupa1/Kacper_Grzymkowski/WUMPD2.ipynb

```diff
@@ -0,0 +1,1993 @@
+{
```

Comment:
<p>To prawda, że 10 iteracji może prowadzić do mało stabilnych wyników - dobrze, że Pan to zauważył i zwiększył liczbę powtórzeń eksperymentu. </p><p>Dobrze, że znalazł Pan workaround dla problemów wydajnościowych.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/105/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>